modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ExtensionMethods.cs

```diff
@@ -221,6 +231,15 @@ private static void FullQualifiedSyntax(SyntaxNode node, SemanticModel sm, Strin
                     sb.Append(child.GetTrailingTrivia());
                 }
             }
+
+            return;
+
```

Comment:
No need to return here, since this is the end of the method.

```suggestion

```

Reply:
Ah yes, this is a common way to explicitly separate the body of the function from the definitions below. Old habits die hard, I'll remove it.

Reply:
It's just that we don't follow that pattern anywhere else where we're already using local methods. But if this is a pattern you feel we should adopt, feel free to suggest it in the RC [#dotnet](https://chat.godotengine.org/channel/dotnet) channel so other contributors can chime in.

I personally feel like it's redundant, but I don't feel strongly about it either way.

Reply:
Nope, I wrote it here out of habit more than anything else.